WebContent/public/about.jsp

```diff
@@ -1,130 +1,138 @@
-		Eric Burns,
-		Todd Elvers,
-		Albert Giegerich,
-		Pat Hawks,
-		Tyler Jensen,
-		Wyatt Kaiser,
-		Ben McCune,
-		Muhammad Nassar,
-		CJ Palmer,
-		Vivek Sardeshmukh,
-		Skylar Stark,
-		and
-		Ruoyu Zhang.
-	</p>
-
-	<p>
-		Computer system support and assistance in designing and building
-		the hardware infrastructure was provided by
-		Hugh Brown,
-		Dan Holstad,
-		Jamie Tisdale,
-		and
-		JJ Ulrich.
-	</p>
-
-	<p>
-		In addition to the members of the Advisory Board,
-		the following people have provided useful feedback and input:
-		Clark Barrett,
-		Christoph Benzm&uuml;ller,
-		Armin Biere,
-		David Cok,
-		Morgan Deters,
-		J&uuml;rgen Giesl,
-		Alberto Griggio,
-		Thomas Krennwallner,
-		Jens Otten,
-		Andrei Paskevich,
-		Olivier Roussel,
-		Martina Seidl,
-		Stephan Schulz,
-		Michael Tautschnig,
-		Christoph Wintersteiger,
-		and
-		Harald Zankl.
-	</p>
-</star:template>
+<%@page contentType="text/html" pageEncoding="UTF-8" %>
+<%@page trimDirectiveWhitespaces="true" %>
+<%@taglib prefix="star" tagdir="/WEB-INF/tags" %>
+
+<star:template title="StarExec" css="about">
+
+  <p id="advisory">StarExec is a cross community logic solving service
+    developed at the University of Iowa and the University of Miami
+        under the direction of principal
+    investigators Aaron Stump (Iowa), Geoff Sutcliffe (University
+    of Miami), and Cesare Tinelli (Iowa).
+  </p>
+
+  <p>Its main goal is to facilitate the experimental evaluation of logic
+    solvers, broadly understood as automated tools based on formal
+    reasoning. The service is designed to provide a single piece of storage
+    and computing infrastructure to logic solving communities and their
+    members. It aims at reducing duplication of effort and resources as well
+    as enabling individual researchers or groups with no access to
+    comparable infrastructure.
+  </p>
+
+  <p>StarExec allows </p>
+  <ul>
+    <li>community organizers to store, manage and make available
+      benchmark libraries;
+    </li>
+    <li>competition organizers to run logic solver competitions; and
+    <li>
+      community members to perform comparative evaluations of logic
+      solvers on public or private benchmark problems.
+    </li>
+  </ul>
+
+  <p>See <a href="https://www.starexec.org/starexec/public/machine-specs.txt">here</a> for the machine 
+       specifications for
+    the compute nodes at the University of Iowa, and 
+        <a href="https://starexec.ccs.miami.edu/starexec/public/machine-specs.txt">here</a> for the compute nodes
+        at the University of Miami.</p>
+
+  <p>A virtual machine for StarExec is <a
+      href="https://www.starexec.org/vmimage/">here</a>. It is recommended
+    to upgrade to the latest <a
+        href="http://www.virtualbox.org">VirtualBox</a> for this
+    (confirmed working for version 4.3.24).</p>
+
+  <p>Use of StarExec is bound by the following
+    <a href="terms-of-service.pdf">terms of service</a>.
```

Comment:
We are in favor of making this a PDF instead of a DOC file, but then could you please add the pdf to the pull request so we will have it for the Iowa site, too?  Right now it is not there, so this would be a broken link at present on our site.